### ISSUE TITLE
Delayed createdCallback

### ISSUE THREAD

**Issue by einarq** [WebReflection/document-register-element#4]
Sorry if this is a non-issue, custom elements and this excellent polyfill is quite new (and exciting) to me.

I'm trying to use this in a Backbone app, where each view has its own template, and that template might contain some custom element (the view then also makes sure that the custom element is registered first).

I noticed that when I insert the template with the custom element in the dom, there is a slight delay before the createdCallback even exists on the custom element and I can communicate with the properties of the custom element.
This does not happen in Chrome where registerElement is natively supported. It also happens when using the polyfill from Polymer though.

Any idea why this happens? This makes it a bit harder to work with, and the difference between the native version and the polyfill is a bit worrisome.

Here is a test case:
http://jsfiddle.net/mt06sykv/

Thanks in advance for any reply.

Einar

**Comment by WebReflection** [comment]
this has been resolved through a possible work-around described here:
https://github.com/WebReflection/document-register-element/pull/15#issuecomment-78037076

TL;DR whenver you need synchronous behavior after DOM manipulation, simply trigger a `readystatechange` event, the poly will take care of everything else.
